### PR TITLE
Super Resulotion GAN

### PR DESCRIPTION
**Pull request by husonchen**
I add some scripts to train SRGan.

### CODE REVIEW

**Comment by mli** [comment]
Job PR-615-1 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-615/1/index.html 
Code coverage of this PR:  vs. Master:

File: scripts/gan/srgan/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+## Reproducing SRGAN experiments
+
+![images](pred.png "images ")
+
+**Download DIV2K dataset**
+```bash
+bash ./download_dataset.sh DIV2K_train_HR
```
Comment (zhreshold) [inline scripts/gan/srgan/README.md]: please use a python script instead of bash

Comment (husonchen) [reply scripts/gan/srgan/README.md]: Yes, This is a mistake of the README file.

**Comment by mli** [comment]
Job PR-615-3 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-615/3/index.html 
Code coverage of this PR:  vs. Master:

**Comment by zhreshold** [comment]
Awesome, I will merge it with 24hr

**Comment by zhreshold** [comment]
Thank you, this is merged.

**Pull request merged**